src/AddressSpace.cc

```diff
@@ -1217,6 +1204,27 @@ void AddressSpace::verify(Task* t) const {
   ASSERT(t, kernel_it.at_end() && mem_it == mem.end());
 }
 
+// Just a place that rr's AutoSyscall functionality can use as a syscall
+// instruction in rr's address space for use before we have exec'd.
+extern "C" {
+extern char rr_syscall_addr;
+}
+static void __attribute__((noinline, used)) fake_syscall() {
+#ifdef __i386__
+  __asm__ __volatile__("rr_syscall_addr: int $0x80\n\t"
+                       "nop\n\t"
+                       "nop\n\t"
+                       "nop\n\t"
+                       : "=a"(r)
+                       : "a"(SYS_geteuid32));
```

Comment:
Nope, I just copied this code from one of the tests and forgot to remove it ;). Will fix.